examples/training/object_detection/pascal_voc/retina_net.py

```diff
@@ -147,28 +147,12 @@ def apply(inputs):
 
 
 def unpackage_inputs(data):
```

Comment:
you could merge this with previous `unpackage_inputs` function to a similar output in the rcnn training script

Reply:
Actually not possible, one format is for the KPLs, one is for the model.  They're distinct; (images, y) => {"images": images, "bounding_boxes": bounding_boxes}

Reply:
(not possible at this time, we could make an OD helper function that unpackages dictionaries?)

Reply:
I'm a little confused....the two training scripts should have unified data input and data output, right?